src/components/features/PairwiseSearch/FeatureInfoPlot.vue

```diff
@@ -0,0 +1,220 @@
+<template>
+    <v-card class="pa-0" style="min-height: 300px; margin: 0px 0px 0px 0px;"
+      outlined>
+      <plotly 
+        ref="chart" 
+        :data="plotData" 
+        :layout="layout" 
+        class="pa-0"
+        style="max-height: 300px;" 
+        >
+      </plotly>
+    </v-card>
+  </template>
+  
+  <script>
+import { Plotly } from "vue-plotly";
+  import PairwiseService from "../../../service/PairwiseService";
+  
+  export default {
+    name: "PathwayResultsPlot",
+    components: {
+      plotly: Plotly,
+    },
+  
+    props: {
+      term: {
+        type: String,
+        default: () => null,
+      },
+    },
+  
+    data() {
+      return {
+        results: [],
+        dataDescriptions: []
+      };
+    },
+  
+    watch: {
+      term() {
+        this.getRelationshipsForGenes();
+      }
+    },
+
+
+    computed: {
+     layout: function () {
+        return {
+          showlegend: true, 
+          responsive: true, 
+          title: {
+            text:'Feature Summary',
+            font: {
+              size: 16
+            },
+          },
+          xaxis: {
+            title: "Sources",
+            showticklabels: false,
+            categoryorder: "array",
+          },
+          yaxis: {
+            title: "Number of " + "<br>" + "Pairwise Features",
+          },
+          hoverlabel: {
+            font: {
+              size: 10,
+            }
+          },
+          modebar: {
+            displayModeBar: true,
+            left: 0.1,
+          }
+        };
+      },
+
+      plotData: function() {
+        return this.createPlotData();
+      }
+    },
+
+    created() {
+      this.getRelationshipsForGenes();
+    },
+  
+    methods: {
+      createPlotData() {
+          let dataDesc2Type = new Map();
+          if(this.dataDescriptions){
+            for(let dataDescription of this.dataDescriptions){
+              let dataType = dataDescription.dataType;
+              if(dataType === "Gene_Coexpression"){
+                let dataResource = dataDescription.id.split("|")[0];
+                dataType += "_" + dataResource;
+              }
+
+              dataDesc2Type.set(dataDescription.id, dataType);
+            }
+          }
+
+          let dataType2pairwiseRelationship = new Map();
+          if(this.results){
+            for(let result of this.results){
+              let descType = dataDesc2Type.get(result.dataDesc.id);
+              if(descType === undefined) descType = "unknown";
+
+              let geneRelationship = dataType2pairwiseRelationship.get(descType);
+              if(geneRelationship === undefined){
+                geneRelationship = {relationshipType: [], posNum: [], text: []};
```

Comment:
relationshipType: better just call dataDescId. It is confusion to call relationshipType since you have used this term for "dataType".